icd/generated/mock_icd.cpp

```diff
@@ -2707,6 +2707,12 @@ static VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceFormatProperties2KHR(
     VkFormatProperties2*                        pFormatProperties)
 {
     GetPhysicalDeviceFormatProperties(physicalDevice, format, &pFormatProperties->formatProperties);
+    VkFormatProperties3KHR *props_3 = lvl_find_mod_in_chain<VkFormatProperties3KHR>(pFormatProperties->pNext);
+    if (props_3) {
+        props_3->linearTilingFeatures = pFormatProperties->formatProperties.linearTilingFeatures;
+        props_3->optimalTilingFeatures = pFormatProperties->formatProperties.optimalTilingFeatures;
+        props_3->bufferFeatures = pFormatProperties->formatProperties.bufferFeatures;
```

Comment:
This does mean that the mock_icd couldn't set bits not found in the 32 bit format properties flags, but that can be left as a todo for later enhancements to mock_icd.

Reply:
Yeah, I think the bottom 32 bits should be the same and in this case is a reasonable assumption to make to get tests working again.